src/opera/instance/node.py

```diff
@@ -91,6 +92,33 @@ def _configure_requirements(self,
                 relationship.run_operation(OperationHost.TARGET, ConfigureInterfaceOperation.shorthand_name(),
                                            target_operation, verbose, workdir)
 
+    def validate(self, verbose, workdir):
+        thread_utils.print_thread("  Validating {}".format(self.tosca_id))
+
+        # validate node's deployment
+        self.run_operation(OperationHost.HOST, StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.CREATE, verbose, workdir)
+        self._configure_requirements(ConfigureInterfaceOperation.PRE_CONFIGURE_SOURCE,
+                                     ConfigureInterfaceOperation.PRE_CONFIGURE_TARGET, verbose, workdir)
+        self.run_operation(OperationHost.HOST,
+                           StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.CONFIGURE, verbose, workdir)
+        self._configure_requirements(ConfigureInterfaceOperation.POST_CONFIGURE_SOURCE,
+                                     ConfigureInterfaceOperation.POST_CONFIGURE_TARGET, verbose, workdir)
+        self.run_operation(OperationHost.HOST, StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.START, verbose, workdir)
+
+        # validate node's undeployment
+        self.run_operation(OperationHost.HOST, StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.STOP, verbose, workdir)
+        self.run_operation(OperationHost.HOST, StandardInterfaceOperation.shorthand_name(),
+                           StandardInterfaceOperation.DELETE, verbose, workdir)
+        self.reset_attributes()
+        self.write()
+
+        self.validated = True
+        thread_utils.print_thread("  Validation of {} complete".format(self.tosca_id))
+
```

Comment:
It's likely I'm blind, but where is the validation actually enabled in this function?

Reply:
Oh, I forgot to push the changes, now they are present.